### PR TITLE
Add config flow to System Monitor

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Adds config flow to System Monitor

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104906
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/systemmonitor.markdown [hunk 1]
```diff
@@ -14,69 +15,49 @@ ha_integration_type: integration
...
+- Packets out - (One per network interface)
+- Network throughput in - (One per network interface)
+- Network throughput out - (One per network interface)
+
+### Other
+
+- Last boot
+- Load (15m)
+- Load (1m)
+- Load (5m)
```
Comment (MartinHjelmare) [inline source/_integrations/systemmonitor.markdown]:
```suggestion
- Load (5m)
- Load (1m)
```

File: source/_integrations/systemmonitor.markdown [hunk 2]
```diff
@@ -14,69 +15,49 @@ ha_integration_type: integration
...
+- Processor use
+- Processor temperature
+- Swap free
+- Swap use
+- Swap use (percent)
 
-If no path is provided via the optional argument, the integration defaults to '/' (root).
+### Add `process` sensor
+
+The `process` sensor needs to be configured by the config entry options. Go to **{% my integrations title="Settings > Devices & Services" %}**, select the **System Monitor** integration and click **Configure** to add or remove `process` sensors.
```
Comment (MartinHjelmare) [inline source/_integrations/systemmonitor.markdown]:
```suggestion
The `process` sensor needs to be configured by the config entry options. Go to **{% my integrations title="Settings > Devices & Services" %}**, select the **System Monitor** integration and click **Configure** to select which `process` sensors should be created.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/systemmonitor.markdown [hunk 3]
```diff
@@ -14,69 +15,49 @@ ha_integration_type: integration
 The `systemmonitor` sensor platform allows you to monitor disk usage,
```
Comment (MartinHjelmare) [inline source/_integrations/systemmonitor.markdown]:
```suggestion
The System monitor integration allows you to monitor disk usage,
```

File: source/_integrations/systemmonitor.markdown [hunk 4]
```diff
@@ -6,77 +6,58 @@ ha_category:
...
+- Packets out - (One per network interface)
+- Network throughput in - (One per network interface)
+- Network throughput out - (One per network interface)
+
+### Other
+
+- Last boot
+- Load (15m)
+- Load (5m)
+- Load (1m)
```
Comment (c0ffeeca7) [inline source/_integrations/systemmonitor.markdown:46]: I see that other sites use m in this context as well. But would it be wrong to use min instead?

**Review by MartinHjelmare — APPROVED** [review]

File: source/_integrations/systemmonitor.markdown (hunk 4 above)
Comment (MartinHjelmare) [reply source/_integrations/systemmonitor.markdown:46]: The "m" means minute so it wouldn't be wrong. Currently the sensors are named "Load (15m)" etc. in core. We should probably be consistent.

https://github.com/home-assistant/core/blob/fbcb31b1030e51cab2aac95484c4008fab40fbe2/homeassistant/components/systemmonitor/sensor.py#L123

I suggest the renaming to be done separately.

**Pull request merged**